peppy/project.py

```diff
@@ -472,7 +412,7 @@ def load_samples(self):
             self[SUBSAMPLE_DF_KEY] = None
 
         for _, r in self[SAMPLE_DF_KEY].iterrows():
-            samples_list.append(Sample(r.dropna(), prj=self))
+            samples_list.append(Sample(r, prj=self))
```

Comment:
@rafalstepien are you sure that here shouldn't be dropna?

Reply:
I'm sure :)